### ISSUE TITLE
Extend more than one 'other' language

### ISSUE THREAD

**Issue by HyunggyuJang** [laishulu/emacs-smart-input-source#30]
Hi, I appreciate your work! It really made writing routines straightforward. 
However, I'd like to extend your package to support more than one other language. Your implementation is useful as it is, but I started to imagine how less cumbersome my life is going to be if I don't ever press input source change command in macOS. As you know, macOS has some issues about CJKV input methods, and I'm using two of them in my workflow -- English, Korean, Japanese. 
With your contribution, I was able to get rid of pressing input source changing command to Korean. Now I hope I'm able to do so for Japanese. 

Again, thanks for this great work!

**Comment by laishulu** [comment]
I have not considering this scenario from the beginning,
but it's possible although nontrivial.

**Comment by HyunggyuJang** [comment]
Yeah, I was tinkering over this for a while. However, as soon as I saw the code, I realized it would be a major change in code base and even if I have done it correctly, my work would break once you upgrade this package. Furthermore, I assume you have your own taste and philosophy on coding and mine might be not in line with yours. So I thought it would be more wise to open an issue hoping you can give it a thought.

Thanks for your attention! Once it has done, this improvement will alleviate a lot of burden for people like me out there who have to press input source toggling commands several times and even have to inspect what the current source is to be sure.
